hack/release-helper.sh

```diff
@@ -22,6 +22,8 @@ fi
 version=$1
```

Comment:
While at it, could you move this into the (already existing) `scripts/hack` directory where other similar scripts are ?

Reply:
yes